01week/rockPaperScissors.js

```diff
@@ -7,17 +7,50 @@ const rl = readline.createInterface({
   output: process.stdout
 });
 
+  // Need a condition to check if hand1 && hand2 have inputs "truthy"
+  // Need a conditions to trim spaces, convert all letters to lowercase, and convert to string
+  // Need a condition to check if the inputs are valid for hand1 && hand2. Does it match 'rock', 'paper', or 'scissors'? If not, return "Please choose type 'rock', 'paper', or 'scissors'."
+  // else if (hand1 === hand2) then return "It's a tie!"
+  // else if (hand1 === 'paper' && hand2 === 'rock') return "Hand one wins!"
+  // else if (hand1 === 'scissors' && hand2 === 'paper') return "Hand one wins!"
+  // else if (hand1 === 'rock' && hand2 === 'scissors') return "Hand one wins!"
+  // else return "Hand two wins!"
 
-function rockPaperScissors(hand1, hand2) {
 
-  // Write code here
+const rockPaperScissors=(hand1, hand2)=>{
 
+  // Check if inputs are "Truthy"
+  if (hand1 && hand2){
+  } else {
```

Comment:
this empty if statement doesn't read well, I would do 
```javascript
if(!hand1 || ! hand2) return ....
```

Reply:
Ah, good catch. Thanks!